climt/_components/surface_ice.py

```diff
@@ -112,29 +80,29 @@ def __init__(self,
         self.quantity_descriptions['default_value'] = CubicSpline(
```

Comment:
Shouldn't this be quantity_descriptions[<quantity name>]['default_value']?

Reply:
yikes. yes.